contracts/kernel/organs/AccountingApp.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity ^0.4.8;                                                         
+
+contract AccountingApp {
+    
+    // The concept of sending tokens from one address to another
+    struct Transaction {
+        address token;
+        int value;
+        address baseToken;
+        address baseValue;
+        
+        address from;
```

Comment:
Either from or to are always the DAO address. We could save some gas just saving a `senderOrRecipient` address field (or a better name) and then a boolean that indicates what is the transaction direction.